libpod/container_inspect.go

```diff
@@ -333,6 +333,8 @@ func (c *Container) generateInspectContainerConfig(spec *spec.Spec) *define.Insp
 		ctrConfig.Umask = c.config.Umask
 	}
 
+	ctrConfig.ProcfsOpts = c.config.ProcfsOpts
```

Comment:
We should just be able to locate the container's procfs mount and copy the options from it.